index.bs

```diff
@@ -1306,6 +1307,11 @@ is true and whose same-site-flag is "`None`"; there is no way to prevent the
 last-access-time from being modified; and the return value is a string that
 would have to be further processed to check for the "`ar_debug`" cookie.
 
+<h3 algorithm id="obtaining-context-origin">Obtaining context origin</h3>
+
+To obtain the <dfn for=node>context origin</dfn> of a [=node=] |node|, return |node|'s [=node navigable=]'s
+<a spec=fenced-frame for=navigable>unfenced container document</a>'s [=origin=].
```

Comment:
We use the term `context origin` elsewhere in the spec. We may want to use something more specifically tied to fenced frame if this dfn only applies to the fenced-frame logic.

Reply:
Yeah I see context origin used in the spec, and I think this algorithm applies to those cases as well (we need the unfenced top-level origin for all API surfaces), therefore adding this as an algorithm.

Reply:
In that case, I think we should make that clear in this PR by updating the other algorithms to use this new dfn.

Reply:
Updated and removed the issue, PTAL, thanks.

Reply:
Actually need to check whether it's suitable. Please wait a bit.

Reply:
Updated, PTAL, thanks!